Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json

```diff
@@ -662,10 +662,139 @@
                 "Users": {
                     "simple": "behaviors.user_name"
                 },
-                "severity": {
-                    "simple": "fine_score"
-                }
-            }
+				"name": {
+					"complex": {
+						"filters": [],
+						"root": "incident_type",
```

Comment:
There is no other way to properly perform this without appending incident type with hostnames and mitre info?

Reply:
There are a million ways to do this
But as the solution works and accepted by CS then why not